common/stats/stats.go

```diff
@@ -1,139 +1,59 @@
-// Copyright 2016 Iron.io
```

Comment:
It's an open repo, and all other files contain this license info (please refer to https://github.com/iron-io/runner/pull/10).
I'd like to retain license info here.